src/components/Toast/Toast.module.css

```diff
@@ -1,170 +1,46 @@
-
 /*------------------------------------*\
     # TOAST
 \*------------------------------------*/
 
 /**
  * 1) Message of information, success, caution, or warning to the user
- * 2) The thick left border operates as a foreground band of color, 
+ * 2) Container for the toast content, to provide background styling
+ * 3) The thick left border operates as a foreground band of color, 
  *    so foreground rather than border is used
  */
-.toast {
-  display: flex;
-  align-items: flex-start;
-  position: relative;
-  padding-top: var(--eds-size-2);
-  padding-right: var(--eds-size-2);
-  padding-bottom: var(--eds-size-2);
-  padding-left: var(--eds-size-2);
-  margin-bottom: var(--eds-size-2);
+ .toast { /* 2 */
+  width: 100%;
   max-width: 25rem;
-  border-style: solid;
-  border-width: var(--eds-theme-border-width);
-  border-color: var(--eds-theme-color-neutral-subtle-border);
-  border-left-width: var(--eds-border-width-lg);
-  border-left-color: var(--eds-theme-color-neutral-medium-foreground); /* 2 */
-  border-radius: var(--eds-theme-border-radius);
-  background-color: var(--eds-theme-color-neutral-subtle-background);
-  box-shadow: var(--eds-theme-box-shadow);
-}
-
-/**
- * Toast success
- * 1) The thick left border operates as a foreground band of color, 
- *    so foreground rather than border is used
- */
-.toast--success {
-  background: var(--eds-theme-color-utility-success-background);
-  border-color: var(--eds-theme-color-utility-success-border);
-  border-left-color: var(--eds-theme-color-utility-success-foreground); /* 1 */
-  color: var(--eds-theme-color-utility-success-foreground);
-}
-
-/**
- * Toast warning
- */
-.toast--warning {
-  background: var(--eds-theme-color-utility-warning-background);
-  border-color: var(--eds-theme-color-utility-warning-border);
-  border-left-color: var(--eds-theme-color-utility-warning-foreground); /* 1 */
-  color: var(--eds-theme-color-utility-warning-foreground);
-}
 
-/**
- * Toast error
- */
-.toast--error {
-  background: var(--eds-theme-color-utility-error-background);
-  border-color: var(--eds-theme-color-utility-error-border);
-  border-left-color: var(--eds-theme-color-utility-error-foreground); /* 1 */
-  color: var(--eds-theme-color-utility-error-foreground);
-}
+  padding-left: var(--eds-size-4);
+  padding-right: var(--eds-size-1);
+  
+  display: grid;
+  gap: var(--eds-size-1);
+  grid-template-columns: 1fr min-content;
+  align-items: center;
+  
+  border: var(--eds-border-width-sm);
+  border-left-width: var(--eds-border-width-lg); /* 3 */
+  border-radius: var(--eds-size-half);
 
-/**
- * Brand toast
- */
-.toast--brand {
-  background: var(--eds-color-brand-blue-100); /* TODO */
-  border-color: var(--eds-theme-color-primary-border);
-  border-left-color: var(--eds-theme-color-primary-foreground); /* 1 */
-  color: var(--eds-theme-color-primary-foreground);
+  border-color: var(--border-light-color);
+  border-left-color: var(--border-dark-color);
```

Comment:
I think these should use a tier-2 token instead, like their variants:
```
  border-color: var(--eds-theme-color-utility-success-border);
  border-left-color: var(--eds-theme-color-utility-success-foreground); /* 3 */
```

also think the 3rd comment refers to color, not width